pandas/tests/reshape/merge/test_merge_anti.py

```diff
@@ -0,0 +1,503 @@
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas import (
+    Categorical,
+    DataFrame,
+)
+import pandas._testing as tm
+from pandas.core.reshape.merge import merge
+
+
+class Test_AntiJoin:
```

Comment:
I think this can all be done in the existing `test_merge.py` file for now

Reply:
I created a new file, because there were separate files for `asof` and `cross`.
Let me know if you feel strongly about this, will paste this into `test_merge.py`